app/views/partials/personal_key/_key.slim

```diff
@@ -7,7 +7,7 @@
     = t('users.recovery_code.header')
   .navy.monospace
     - code.split(' ').each do |word|
-      p.bold.mb0.h3 data-recovery='word'
+      p.bold.mb0.h3(data-recovery="#{word}")
```

Comment:
Annoyingly, trim is also actually only available >= IE 9, but we can replace the white space, at least.